README-snap.md

```diff
@@ -0,0 +1,185 @@
+# openstack-exporter snap
```

Comment:
Move this file under snap/README.md

Reply:
It's a pity that `snapcraft` does not have `.snapcraft.ignore` or such.

The following message when `snapcraft pack` should be harmless
```
The 'snap' directory is meant specifically for snapcraft, but it contains
the following non-snapcraft-related paths:
- README.md

This is unsupported and may cause unexpected behavior. If you must store
these files within the 'snap' directory, move them to 'snap/local'
which is ignored by snapcraft.
```